includes/ServiceWiring.php

```diff
@@ -29,6 +30,18 @@
 		);
 	},
 
+	'WikibaseReconcileEdit.EditRequestSaver' => function ( MediaWikiServices $services ): EditRequestSaver {
+		$repo = WikibaseRepo::getDefaultInstance();
+		$editEntityFactory = method_exists( $repo, 'getEditEntityFactory' )
+			? $repo->getEditEntityFactory() // 1.36+
```

Comment:
Hm, in the 1.36 version we could actually pass `$services` into this method, to ensure that the edit entity factory comes from the same service container. But I’m not sure if that’s very important here.

Reply:
okay